client/app/components/Select.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useRef, useState, useEffect } from "react";
 import { sortBy, get } from "lodash";
 import AntdSelect, { SelectProps } from "antd/lib/select";
 
-function getItemOfPercentile<T>(list: Array<T>, percentile: number, sortIteratee: string | string[] = "length") {
+function getItemOfPercentile<T>(list: Array<T>, percentile: number, sortIteratee = "length") {
```

Comment:
String array will break get

Reply:
It shouldn't, these two examples are totally equal: `_.get(obj, "path.to.prop")` and `_.get(obj, ["path", "to", "prop"])`. But I may guess that the actual reason is that string array works differently with `_.get` and `_.sortBy` 🙂